src/Perspectives/unknown/FactoryPerspective.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+pragma solidity ^0.8.24;
+
+import {BasePerspective} from "../BasePerspective.sol";
+
+contract FactoryPerspective is BasePerspective {
+    constructor(address vaultFactory_) BasePerspective(vaultFactory_) {}
+
+    function name() public pure virtual override returns (string memory) {
+        return "Unknown.Unknown.FactoryPerspective";
```

Comment:
Another case where the immutable/governed classification doesn't fit.

Reply:
Yes. Although this perspective is only for UI testing purposes. In practice, it would never be used.